handler.go

```diff
@@ -49,12 +51,19 @@ type item struct {
 	Comments []comment `json:"-"`
 }
 
+type User struct {
+	ID        bson.ObjectId `json:"id" bson:"id"`
+	Name      string        `json:"name" bson:"name"`
```

Comment:
bson by default is lowercase so you don't need to define it unless it has an underscore (see item.Type and item.StargazersIDs)

Reply:
fixed!